workloads/mastervertical.yml

```diff
@@ -108,10 +106,20 @@
       shell: |
         oc replace --force -n scale-ci-tooling -f "{{ansible_user_dir}}/scale-ci-tooling/workload-job.yml"
 
-    # Might need retires to account for api server cert rotation / connection breaking
-    # Wait for job start, then wait for complete
-    # oc get job scale-ci-mastervertical -n scale-ci-tooling -o json | jq '.status.active'
-    # oc get job scale-ci-mastervertical -n scale-ci-tooling -o json | jq '.status.succeeded'
-    # - name: Wait for job to complete
-    #   shell: |
-    #     oc logs -f job/scale-ci-mastervertical -n scale-ci-tooling
+    - name: Poll (2m) until job is active
```

Comment:
I suggest dropping the (2m).  60*2 is 120s, but the `oc` command usually takes a couple of seconds to return a result itself, therefore in reality this can be a lot more than 2m.